src/components/crypto/WalletConnect.js

```diff
@@ -104,7 +155,24 @@ const WalletConnect = observer(({type="phantom", showPaymentPreference, onConnec
               { wallet.networkName } { rootStore.l10n.connected_accounts.wallet_address }
             </div>
             <div className="wallet-connect__network-address ellipsis" title={connectedAccount.link_acct}>
-              { connectedAccount.link_acct }
+              {
+                type === "circle_acct" ?
+                  <div>
+                    <div className="wallet-connect__help-link">
+                      { rootStore.client.networkName === "main" ?
+                        <a href= {"https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48?a=" + rootStore.cryptoStore.CircleLinkedAddress()}  target="_blank" rel="noopener">
```

Comment:
can this etherscan (and goerli below) contract address live in config somewhere or be fetched?  how will we change or update it?

Reply:
I don't see anything but EluvioConfiguration configuration.js which isn't checked in.   I guess I can make a src/utils/USDCExplorer.js for the base info and then it can be overridden from EluvioConfiguration